cf/i18n/resources/ja-jp.all.json

```diff
@@ -1253,7 +1253,7 @@
   },
   {
     "id": "Creating an app manifest from current settings of app ",
-    "translation": "アプリの現在の設定からアプリ・マニフェストを作成しています"
+    "translation": "アプリの現在の設定からアプリ・マニフェストを作成しています "
```

Comment:
It is not common in Japanese to add a space at the end of a sentence. Why this change?